WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/NewSimplePaymentsLocationNoticeViewModel.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+final class NewSimplePaymentsLocationNoticeViewModel {
+    /// Redirects to `HubMenu`tabBar
+    ///
+    var navigateToMenuButtonWasTapped: () -> Void = {
+        guard let mainTabBarController = AppDelegate.shared.tabBarController else {
+            return
+        }
+        mainTabBarController.navigateTo(.hubMenu)
+    }
+    /// Uses the "woo-payments-wallet" UIImage+Woo as icon
+    ///
+    var icon: UIImage {
+        guard let image = UIImage(named: "woo-payments-wallet") else { return UIImage() }
+        return image
+    }
+    let title: String
+    let message: String
+    let confirmationButtonMessage: String
+
+    init() {
```

Comment:
It would be nice if we follow here the usual order for a class elements:

- stored properties
- computed properties
- init
- public functions
- private functions (in a private extension)